Makefile

```diff
@@ -46,11 +46,11 @@ coverage:
 		coverage html
 
 container-deploy:
-	docker build -t modulus:deploy --target deploy -f Dockerfile .
+	docker build -t modulus:deploy --build-arg TARGETPLATFORM=linux/amd64 --target deploy -f Dockerfile .
```

Comment:
so how do you build with buildx? is that documented somewhere? maybe the release repo

Reply:
I'm not familiar with the buildx builder and what not. Just some info for knowledge transfer somewhere even if its internal would be good to capture.

Reply:
Yes, I had captured that in the PR description :). But will also document it in the release repo. The regular builds will work just fine for single arch image.